techlibs/gowin/synth_gowin.cc

```diff
@@ -280,6 +280,8 @@ struct SynthGowinPass : public ScriptPass
 			run("opt_lut_ins -tech gowin");
 			run("setundef -undriven -params -zero");
 			run("hilomap -singleton -hicell VCC V -locell GND G");
+			if (!vout_file.empty() || help_mode) // vendor output requires 1-bit wires
```

Comment:
note that this conition will unconditionally show this line in help mode without any annotation; change the following line to something like this:

`run("splitnets -ports", "(only if -vout used)");`